source/content/guides/custom-upstream/02-create-custom-upstream.md

```diff
@@ -56,8 +56,6 @@ Follow the steps for your preferred Git host below to create your repository.
 
   As a company, Pantheon is trying to use [more inclusive language in our repositories](https://pantheon.io/blog/diversity-equity-and-inclusion-pantheon?docs). While our team works to make Custom Upstreams less reliant on older naming conventions, new Custom Upstreams currently default to using the `master` branch name.
 
-  Git's default naming convention differs from GitHub's. If you don't initialize the repository on GitHub, Git will assign the default branch name as `master` when you clone the repository locally.
```

Comment:
Note for docs: the deleted text was true for old versions of git, but current versions now also default to `main` for the initial default branch.